apps/pie-storybook/stories/pie-link.stories.ts

```diff
@@ -88,16 +93,19 @@ const linkStoryMeta: LinkStoryMeta = {
 export default linkStoryMeta;
 
 const Template = ({
-    href, target, rel, size, variant, isBold, isStandalone, slot,
+    href, target, rel, size, variant, isBold, isStandalone,
+    slot, iconPlacement,
 }: LinkProps): TemplateResult => html`
         <pie-link
             variant="${variant}"
             size="${size}"
+            iconPlacement="${iconPlacement || nothing}"
             href="${href || nothing}"
             target="${target || nothing}"
             rel="${rel || nothing}"
             ?isBold="${isBold}"
             ?isStandalone="${isStandalone}">
+            ${iconPlacement ? html`<icon-plus-circle slot="icon"></icon-plus-circle>` : ''}
```

Comment:
I think we do this in some other components?
```suggestion
            ${iconPlacement ? html`<icon-plus-circle slot="icon"></icon-plus-circle>` : nothing}
```

Reply:
fixed